components/CountryNavigator.js

```diff
@@ -0,0 +1,248 @@
+    color: " #cbd5e1",
+    "&:hover": {
+      borderColor: "#f8fafc",
+      color: "#f8fafc",
+      boxShadow: "none",
+    },
+    "&:active": {
+      boxShadow: "none",
+      color: "#f8fafc",
+      borderColor: "#f8fafc",
+    },
+    "&:focus": {
+      boxShadow: "0 0 0 0.2rem rgba(0,123,255,.5)",
+    },
+  }));
+  const colorScale = scaleOrdinal({
+    domain: CATEGORIES_ORDERED_LIST,
+    range: RAINBOW_COLORS,
+  });
+
+  return (
+    <div className={`w-full mx-auto `}>
+      <Head>
+        <title>{`${
+          countries[currentCountryIndex] ?? "Loading"
+        } - Flags of Inequality`}</title>
+      </Head>
+      <h2 className="sr-only">{`Timeline of ${countries[currentCountryIndex]} from 2015 to 2024`}</h2>
+      <div className={`flex  gap-4 mt-10`} ref={allFlagsContainer}>
+        {allButLast.map((year) => (
+          <div
+            key={`${countries[currentCountryIndex]}-${year}`}
+            className={`flex flex-col gap-4 mt-4`}
+            layout
+          >
+            <motion.div key="flag">
+              <Box
+                sx={{
+                  width: allFlagDimensions.width,
+                  height: allFlagDimensions.height,
+                }}
+              >
+                <Flag
+                  country={countries[currentCountryIndex]}
+                  year={year}
+                  isTimeline={"true"}
+                  orientation={"horizontal"}
+                  isInteractive={false}
+                />
+              </Box>
+            </motion.div>
+            {currentCountryData.find((d) => d.year === year)?.ranking !==
+            undefined ? (
+              <div className="flex gap-2 ">
+                <h3 aria-hidden>
+                  {" "}
+                  {year}{" "}
+                  <span className="text-slate-400">{` | ${nth(
+                    currentCountryData.find((d) => d.year === year)?.ranking
+                  )}`}</span>
+                </h3>
+                <h3></h3>
+              </div>
+            ) : null}
+          </div>
+        ))}
+      </div>
+      <div className={`flex mt-10 gap-4 visible`} ref={flagsContainer}>
+        {lastYear.map((year) => (
+          <div key={`${countries[currentCountryIndex]}-${year}`}>
+            <motion.div key="flag">
+              <Box
+                sx={{
+                  width: flagDimensions.width,
+                  height: flagDimensions.height,
+                }}
+              >
+                <Flag
+                  country={countries[currentCountryIndex]}
+                  year={year}
+                  isTimeline={"false"}
+                  orientation={"horizontal"}
+                  isLastYear={"true"}
+                  isInteractive={false}
+                />
+              </Box>
+            </motion.div>
+          </div>
+        ))}
+        <div style={{ display: "inline-block", alignSelf: "flex-end" }}>
+          <p style={{ color: colorScale("equality") }}>
```

Comment:
nit: Maybe we can take a similar approach to the `<Legend>` component and use the `CATEGORIES_FULL_NAME` array and the `CATEGORIES` object, so we can wrap this nicely in a map?